docs/development/install.rst

```diff
@@ -184,3 +184,37 @@ For building this documentation,
 see :doc:`docs`.
 
 And for setting up for the front end development, see :doc:`standards`.
+
+Troubleshooting
+---------------
+
+You might face issues while installing.
+
+Error: `SocialApp matching query does not exist.`
+
+Please follow the guidelines for installation given here:
+
+- Start the installation
+  Run the installation: `sudo python3 manage.py runserver 0.0.0.0:<number>`
+
+- Visit the readthedocs DJANGO Admin site:
+  `http://localhost:<number>/admin`
+
+- EDIT the SITES section:
+  In the sites section of the Django Admin panel change the example.com domain as
+  the localhost admint/sites domain with: `http://localhost:<number>/admin/sites`
+
+**Note:** The ID of the site should be of the `SITE_ID` key in `readthedocs/settings/base.py`.
+
+- Connect GitHub account
+
+- You should create a test [GitHub OAuth Apps](https://github.com/settings/applications/new).
+
+- Add a social account in the readthedocs installation via Django admin panel
+  - visit `http://localhost/admin/socialaccount/ and tab on a social account`.
+  - Write in the tab saying `GitHub`
+  - Add `availble site` and `chosen site`.
```

Comment:
We may want to tell them to collect the Client id and Secret key from the OAuth App and add it here.